### DISCUSSION TITLE
new category: translations?

### DISCUSSION THREAD

**Discussion by axd1967** [Stellarium/stellarium#1689] [Ideas]
Maybe we need another discussion category: Translations?

As far as I can see, Transifex does not offer forums for discussions.

See also https://github.com/Stellarium/stellarium/wiki/Translation-of-Stellarium which refers to a general https://groups.google.com/forum/#!forum/stellarium (to be honest, I didn't follow that group; I joined it just now). I'm tempted to propose to archive that Google group in order to better bundle resources, or maybe redefine the purpose of that Google group? Or at least redirect translators to this new discussion category?

And I guess there exist more forums out there, maybe on SourceForge; these should still be reachable (and therefore referenced somewhere) but archived for as long as they exist.

**Comment by gzotti** [comment]
We don't need more discussion. We can use a few more qualified developers who solve issues, not create them.

**Reply by axd1967** [reply]
@gzotti I hope you understand that I cannot accept being accused that way in public, because that's the jist of your phrase. This behaviour has been going on in many places in this project, and I do believe this is all based on misunderstandings and wrong translations.

I have ALWAYS been nice, and if you can point me to rude language anywhere, please do so now, otherwise I kindly ask you to stop this behaviour. I have already contributed code and do so within my resources. I also contribute as translator. I strongly believe that contributions are more than by code alone: that includes opening issues, asking questions, helping users, suggesting project improvements, suggesting programming and code versioning patterns, and much more.

(@treaves @Atque )

**Reply by axd1967** [reply]
it is perfectly understandable that Stellarium wants more developer resources, but a lack of developers is not a reason to forbid other types of inputs, as long as these are constructive.

**Reply by Atque** [reply]
I'll have to agree with @axd1967 here. The tone here is sometimes unnecessarily harsh, and that is not how to attract new developers to a project. Everyone is here because of interest in astronomy, and any kind of contribution, be it through actual code or pointing out errors or possible improvement, should be welcome and encouraged.

**Comment by axd1967** [comment]
See https://github.com/Stellarium/stellarium/discussions/categories/translations
